lib/Elevate/Roles/Run.pm

```diff
@@ -74,7 +74,7 @@ sub _ssystem ( $command, %opts ) {
     my @args = @{ $command // [] };
 
     # Only allow the program to be an executable absolute path
-    return 42 if $args[0] !~ '^/' || !-x $args[0];
+    die "Program is not an executable absolute path." if $args[0] !~ '^/' || !-x $args[0];
```

Comment:
Let's use `Carp::confess` so that we get a backtrace.

Reply:
Agreed this initial commit was extremely literal in it's use of die().